docs/wp/economics.md

```diff
@@ -63,7 +63,7 @@ Mining reward is capped at 25 920 iDNA per day. It includes block proposer rewar
 | ----------------------------------- | ------------------ |
 | Block proposer reward cap per day   | 8 640 iDNA (~33%)  |
 | Block committee reward cap per day  | 17 280 iDNA (~67%) |
-| Minimum block time                  | 12 sec             |
+| Minimum block time                  | 10 sec             |

```

Comment:
The minimum block time is 20 sec. Maximum number of blocks per minute is 3.

Reply:
The maximum block time is 20 seconds since there can't be more than 3 blocks per minute.
But in the source code, the validateBlockTimestamp would only deny a block if the time between blocks is less than 10 (MinBlockDelay). Is this used for something different?
Reference: https://github.com/idena-network/idena-go/blob/master/blockchain/blockchain.go#L2286

Reply:
I've updated this. I believe that means the source code has a MinBlockDelay of 10 seconds to give some space for time differences between nodes.